skbio/embedding/_embedding.py

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import numpy as np
+
+from skbio.sequence import Sequence
+from skbio._base import SkbioObject
+from skbio.metadata._mixin import MetadataMixin
+
+
+class Embedding(SkbioObject):
+    """Store embeddings for a biological object."""
+
+    @property
+    def embedding(self):
+        return self._embedding
+
+    @property
+    def ids(self):
```

Comment:
Oh I see, this is an `id` in the sense that an ASV is an `id`. The naming is a bit confusing, as often IDs do not have meaning but are for establishing linking relationships. Should this be renamed to `peptides` or `sequences`?

Reply:
Yes we can add a `residues` method to make this more transparent on a per object basis. But maybe we should have an `id` method in the `Embedding` ABC so that all subclasses can still be queried with the same attribute?